src/AngleSharp.Scripting.JavaScript/AngleSharp.Scripting.JavaScript.csproj

```diff
@@ -87,5 +82,11 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\AngleSharp\src\AngleSharp\AngleSharp.Core.csproj">
```

Comment:
This is a problem for me - how do you develop code for this branch?? I mean 0.10.0.0 doesn't exist in the nuget repo so the build fails...